pavelib/paver_tests/test_xsscommitlint.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Did the history get lost on these renames, or is this just GitHub getting confused? It would be best to see these as true renames.